chapters/kmodes/main.tex

```diff
@@ -738,125 +785,127 @@ \subsection{Using number of classes for \(k\)}\label{subsec:nclasses}
+All of the results leading up to this point were created using benchmark
+datasets. The discussion at the start of Chapter~\ref{chp:edo} does not condemn
+this entirely as there are certainly benefits to comparing methods in this way
+such as familiarity and direct comparison. However, as that discussion
+elaborates on, it does not afford a rich understanding of how any of the methods
+perform more generally. This stage of the analysis relies on the method for
+generating artificial datasets introduced in Chapter~\ref{chp:edo}, and, as was
+made clear in that chapter's case study, the critical component of this method
+is a well-defined fitness function.
+
+In order to expose the nuances in the performance of Cao's method and the
+proposed initialisation on a particular dataset, two cases are considered: where
+Cao's method outperforms the proposed, and vice versa. Both cases use the same
+fitness function, although the latter uses its negative. The function is defined
+as follows:
 \begin{equation}\label{eq:fitness}
     f\left(\mathcal X\right) = C_{\mathrm{cao}} - C_{\mathrm{match}}
 \end{equation}
-where \(C_{\mathrm{cao}}\) and \(C_{\mathrm{match}}\) are the final costs when a
+
+Here, \(C_{\mathrm{cao}}\) and \(C_{\mathrm{match}}\) are the final costs when a
 dataset \(\mathcal X\) is clustered using Cao's method and the proposed matching
-method respectively with \(k = 3\). For the sake of computational time, the
-proposed initialisation was given 25 repetitions as opposed to the 250
-repetitions in the remainder of this section. Apart from the sign of \(f\), the
-dataset generation processes used identical parameters in each case and the
-datasets considered here are all of comparable shape.
+method respectively with \(k = 3\). Given that costs should be minimised, this
+function is given to EDO to be minimised. As such, the case that uses \(f\) will
+be referred to as Cao-preferable. For the sake of computational time, the
+proposed initialisation is given 25 repetitions.
+
+Apart from the sign of \(f\), each case uses identical EDO parameters: a
+population size of \(N = 500\); row and column limits of \(R = (50, 500)\) and
+\(C = (2, 50)\), respectively; selection parameters \(b = 0.2\) and \(l = 0\); a
+mutation probability of \(p_m = 0.01\); a maximum number of iterations of \(M =
+100\). In addition to these parameters, the only distribution family included is
+a discrete uniform family with up to 10 categories. This means that all columns
+in the generated datasets will have at least one category but no more than 10.
 
 \begin{figure}
     \centering
     \includegraphics[width=\imgwidth]{Fig9.pdf}
-    \caption{Histograms of fitness for the top performing percentile in each
-             case.}\label{fig:fitness}
+    \caption{%
+        Histograms of fitness for the top performing percentile in each case.
+    }\label{fig:fitness}
 \end{figure}
 
 This process yielded approximately 35,000 unique datasets for each case, and the
```

Comment:
Can we add a link to the archived data here?

Reply:
Fixed by 80e2164